packages/suite/src/actions/suite/initAction.ts

```diff
@@ -51,7 +51,7 @@ export const init = () => async (dispatch: Dispatch, getState: GetState) => {
     }
 
     // 6. init backends
-    await dispatch(blockchainActions.init()).catch(err => console.error(err));
+    await dispatch(initBlockchainThunk()).catch(err => console.error(err));
```

Comment:
catch will never occur because thunks from RTK are always resolved, you need to add unwrap()

Reply:
https://github.com/trezor/trezor-suite/pull/6140/commits/74d13ed8eca1360367887af42bf58f7ef5bc362e